### PR TITLE
OpenScanHub, Fedora and Packit

### PR DESCRIPTION
**Pull request by siteshwar**

### CODE REVIEW

File: _posts/2024-07-31-openscanhub-fedora-and-packit.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: OpenScanHub, Fedora and Packit
+layout: post
+category: posts
+---
+
+TLDR: We have deployed OpenScanHub on the [Fedora infrastructure](https://openscanhub.fedoraproject.org/) and recently integrated with [Packit](https://packit.dev/). An example scan can be seen on [GitHub](https://github.com/strace/strace/pull/300/checks?check_run_id=28180162251) and more information can be found in the [OpenScanHub](https://fedoraproject.org/wiki/OpenScanHub) and [Packit](https://packit.dev/docs/configuration) documentation.
```
Comment (lbarcziova) [inline _posts/2024-07-31-openscanhub-fedora-and-packit.md:7]:
```suggestion
TLDR: We have deployed OpenScanHub on the [Fedora infrastructure](https://openscanhub.fedoraproject.org/) and recently integrated with [Packit](https://packit.dev/). An example scan can be seen on [GitHub](https://github.com/strace/strace/pull/300/checks?check_run_id=28180162251) and more information can be found in the [OpenScanHub](https://fedoraproject.org/wiki/OpenScanHub) and [Packit](https://packit.dev/docs/configuration#osh_diff_scan_after_copr_build) documentation.
```

File: _posts/2024-07-31-openscanhub-fedora-and-packit.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+# Packit
+Packit is an open source project aiming to ease the integration of projects with Fedora Linux, CentOS Stream and other distributions. Since I started working on OpenScanHub a couple of years ago, Packit has always been on the top of my list for integration. It is used by some very prominent open source projects like systemd, strace, util-linux, etc. and is targeted towards upstream developers to decrease the maintenance burden of packaging. It is a great integration point for OpenScanHub as the bugs should be fixed as soon as possible upstream.
+
+The Packit team did service side integration of OpenScanHub recently. It is enabled only for rawhide, as rawhide is the closest candidate to upstreams. A differential scan is performed on each pull request against the target branch, so the developers can focus only on the findings introduced in the pull requests.
```
Comment (lbarcziova) [inline _posts/2024-07-31-openscanhub-fedora-and-packit.md:19]:
```suggestion
The Packit team did service side integration of OpenScanHub recently. It is enabled only for rawhide, as rawhide is the closest candidate to upstreams. A differential scan is performed on each pull request against the target branch, so the developers can focus only on the findings introduced in the pull requests (see the details of the required setup [here](https://packit.dev/docs/configuration#osh_diff_scan_after_copr_build).
```

File: _posts/2024-07-31-openscanhub-fedora-and-packit.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+Alternatively, you can use the Packit CLI tool to submit a scan. If you are already using Packit, you can use the `packit scan-in-osh` subcommand to perform a scan. This feature is documented through [Packit CLI](https://packit.dev/docs/cli/scan-in-osh) documentation.
```
Comment (lbarcziova) [inline _posts/2024-07-31-openscanhub-fedora-and-packit.md]:
```suggestion
Alternatively, you can use the Packit CLI tool to submit a scan. If you are already using Packit CLI, you can use the `packit scan-in-osh` subcommand to perform a scan. This feature is documented through [Packit CLI](https://packit.dev/docs/cli/scan-in-osh) documentation.
```

**Review by lbarcziova — COMMENTED** [review]
nice! 

Maybe the [feedback discussion](https://github.com/packit/packit/discussions/2371) is also worth mentioning.

File: _posts/2024-07-31-openscanhub-fedora-and-packit.md (hunk 1 above)
Comment (siteshwar) [reply _posts/2024-07-31-openscanhub-fedora-and-packit.md:7]: Is there a chance this field name may change and break the link later? I wanted to avoid breaking links in the blog post.

File: _posts/2024-07-31-openscanhub-fedora-and-packit.md (hunk 2 above)
Comment (siteshwar) [reply _posts/2024-07-31-openscanhub-fedora-and-packit.md:19]: Same as above.

**Comment by siteshwar** [comment]
> nice!
> 
> Maybe the [feedback discussion](https://github.com/packit/packit/discussions/2371) is also worth mentioning.

I wanted to reference the blog post Packit team is preparing, probably users can take it from there. For some reason, I try to keep my blog posts as minimal as possible, specially when it comes to links (that break or become irrelevant over time).

**Comment by siteshwar** [comment]
For some reason, my changes were not reflecting in this pull request. So, I am going to reopen another one.

**Comment by siteshwar** [comment]
https://github.com/siteshwar/siteshwar.github.io/pull/4
